chainweb.cabal

```diff
@@ -572,114 +614,6 @@ executable chainweb-miner
         , strict-tuple >= 0.1
         , wai-extra >= 3.0
 
-executable cwtool
```

Comment:
We shouldn't delete this section completely.  We still need the cwtool executable.  It's just the modules in `other-modules` that we want to re-use that need to be moved into the `library` section.

Reply:
Ok, that makes sense. I'll make that change.